theme/blog-btn.scss

```
@@ -25,10 +25,10 @@
   line-height: 32px;
   padding-left: 43px;
   padding-right: 20px;
-  transition: all 0.3s ease;
+  transition: background 0.3s ease;
   user-select: none;
-  opacity: 0;
-  transition: opacity 0.5s ease-in;
+  // Visible in SSG HTML — do not wait for client JS to reveal the badge.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)